### ISSUE TITLE
Error using Latent mode

### ISSUE THREAD

**Issue by TemUsername** [hako-mikan/sd-webui-regional-prompter#178]
Attention mode works fine but doesn't give the result I'm looking for. I tried using Latent mode with and without any Loras and it throws me this error 
```
Error running process: D:\AI\stable-diffusion-webui\extensions\sd-webui-regional-prompter\scripts\rp.py
Traceback (most recent call last):
  File "D:\AI\stable-diffusion-webui\modules\scripts.py", line 409, in process
    script.process(p, *script_args)
  File "D:\AI\stable-diffusion-webui\extensions\sd-webui-regional-prompter\scripts\rp.py", line 406, in process
    setuploras(self)
  File "D:\AI\stable-diffusion-webui\extensions\sd-webui-regional-prompter\scripts\latent.py", line 64, in setuploras
    lora.lora_forward = lora_forward
NameError: name 'lora_forward' is not defined
``` 

The image I end up getting is just a mess with unrecognizable figures

**Comment by hako-mikan** [comment]
Thanks reporting.
May be fixed.

**Comment by creeduk** [comment]
I have this same issue with Latent. I installed Monday, works great with attention mode but if I leave everything the same and select latent I get a similar issue. I have no Lora selected and I updated it again today but still I get this:

`return lora.lora_forward(self, input, torch.nn.Linear_forward_before_lora)
AttributeError: module 'lora' has no attribute 'lora_forward'`

Now if I switch back to Attention the error continues until I reload the webpage.

If I turn off xformers then it runs 1 time (second time I get error above) but with this error:

`Error running process: C:\Users\AI\stable-diffusion-webui\extensions\sd-webui-regional-prompter\scripts\rp.py
Traceback (most recent call last):
  File "C:\Users\AI\stable-diffusion-webui\modules\scripts.py", line 417, in process
    script.process(p, *script_args)
  File "C:\Users\AI\stable-diffusion-webui\extensions\sd-webui-regional-prompter\scripts\rp.py", line 424, in process
    setuploras(self)
  File "C:\Users\AI\stable-diffusion-webui\extensions\sd-webui-regional-prompter\scripts\latent.py", line 58, in setuploras
    orig_lora_functional = shared.opts.lora_functional
  File "C:\Users\AI\stable-diffusion-webui\modules\shared.py", line 493, in __getattr__
    return super(Options, self).__getattribute__(item)
AttributeError: 'Options' object has no attribute 'lora_functional'`

tried:

--lowvram --api --xformers
&
--lowvram

Same issue, not urgent not sure if it would be useful to use I was just curious what Latent would give.
